### ISSUE TITLE
Metadata textarea doesn't save new lines

### ISSUE THREAD

**Issue by jpeg729** [prose/prose#1033]
If you enter a text with line breaks in a textarea field, prose just saves it as a single line. I find that really annoying, though I can see some use cases.

Can I suggest the addition of a new metadata element type called "markdown" that would let the user enter some formatted text and would save the entered markdown to the yaml field? That would be great.

**Comment by 510interactive** [comment]
I agree.  Though, it might be difficult to get markdown into front matter.   I will probably spend a little bit of time at least trying to get textarea fields to honor line breaks.

**Comment by dereklieu** [comment]
What are you using text areas for that you require line breaks? If you're creating a list/array, a multi-select field would be a better option. If your goal is to create blocks of formatted prose (ie paragraphs), I'm not sure the yaml spec supports that, and it would be better done in jekyll using includes.

**Comment by jpeg729** [comment]
I have a jekyll collection for events, and I would like my client to be able to update these events himself without worrying too much about html tags and so on. Now the problem is that each event must be stored in a single file, an event has a price, a time, and a description, and I want to display these three bits of info in different parts of the web page. I store the price and the time in yaml metadata fields, and the description in the main content.

The problem arises when for a particular event the description of the time or the price is a little more complicated than usual and to fit it nicely into the allotted space you need to add a line break at a specific point. Currently the only way of doing this is to add html tags at the point where you want the line break. But that isn't especially user friendly.

yaml provides the block scalar identifier `|` which would do the job quite nicely for my use case. It would even work for most of markdown.
```yaml
field: |
    # This is some markdown content stored in YAML

    This will be output as a paragraph tag.

    So will this!
```

Even better might be `|-` which keeps all the line breaks except for the trailing ones.

**Comment by jpeg729** [comment]
I have already seen prose store the contents of a textarea element using `>-` which allows you to store strings using multiple lines, but removes the line breaks when reading it back.
